src/components/CountryCard/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import CustomCheckbox from '../customCheckbox/index';
+import './styles.scss'
+
+const CountryCard  = ({country, onChange, deleteHandler, path}) => {
+    return (
+        <Link to={`${path}`} >
+            <div className='card-wrapper'>
+                <img src={country.flag} alt="flag" />
+                <span>{country.name}</span>
+                {country.callingCodes[0] !== '' ? <span>Country code: +{country.callingCodes}</span> : null}
+                <div className='hidden-menu-wrapper'>
+                    <CustomCheckbox 
+                        checked={country.isPinned} 
+                        onChange={() => onChange(country, country.id)}
+                        style={{color: 'black'}}
```

Comment:
inline styles is bad practice, styled components  can apply styles to lib's components